manifests/create-cert.sh

```diff
@@ -14,11 +14,26 @@ detailed explantion and additional instructions.
 
 The server key/cert k8s CA cert are stored in a k8s secret.
 
+       --namespace        Namespace where webhook service and secret reside.
 EOF
     exit 1
 }
 
-namespace=default
+while [[ $# -gt 0 ]]; do
```

Comment:
Refer [here](https://github.com/heroku/heroku-buildpack-php/blob/master/bin/heroku-hhvm-nginx#L141) to implement shell parameter parsing

Reply:
done